tests/util/setup_tests.rs

```diff
@@ -45,9 +45,10 @@ use libstratis::engine::strat_engine::StratEngine;
 /// 6. Verify that get_metadata() return an error. initialize() only
 /// initializes block devices, it does not write metadata.
 pub fn test_initialize(paths: &[&Path]) -> () {
-    assert!(paths.len() > 2);
+    assert!(paths.len() > 1);
 
-    let (paths1, paths2) = paths.split_at(2);
+    let (paths1, paths2) = paths.split_at(paths.len() / 2);
+    assert!(paths1.len() > 0 && paths2.len() > 0);
```

Comment:
Better to get rid of this assert and corresponding asserts below. Othewrise, we're just checking correctness of Rust integer division, which we had better be able to rely on.